### PR TITLE
Add Cryptocat to "Built on Electron" section

### PR DESCRIPTION
**Pull request by kaepora**
[Cryptocat](https://crypto.cat) free software is entirely built on Electron. I'm quite proud of the software's quality, and it wouldn't have been possible without your fantastic framework.

Furthermore, Cryptocat uses a *lot* of Electron's features, including desktop integration and so on.

### CODE REVIEW

**Comment by jlord** [comment]
Awesome! I really like your icon 😸

File: index.html [hunk 1]
```diff
@@ -312,6 +312,9 @@
 
             <a href='https://datazenit.com' target='_blank' title='Datazenit – Modern database administration tool'>
               <img class='other-logo' src='../images/datazenit.png' alt='Datazenit – Modern database administration tool'></a>
+              
+            <a href='https://crypto.cat' target='_blank' title='Cryptocat - Free secure chat software'>
+              <img class='other-logo' src='../images/cryptocat.png' alt='Cryptocat - Fre secure chat software'></a>
```
Comment (jlord) [inline index.html]: Just a small typo here, the "Free" is missing an "e"

**Comment by kaepora** [comment]
@jlord Thanks! I've fixed the typo you pointed out in b66b1801c61e9af6d84426281bd2330c6f204d9c.

File: index.html (hunk 1 above)
Comment (kaepora) [inline index.html]: Fixed in b66b1801c61e9af6d84426281bd2330c6f204d9c.

**Comment by jlord** [comment]
Whoops wrong button 😀

**Pull request merged**